src/main/java/nl/techiteasy/Techiteasy_part2/Services/TelevisionService.java

```diff
@@ -121,4 +125,20 @@ public TelevisionOutputDto makeTheDto (Television t) {
         teleDto.sold = t.getSold();
         return teleDto;
     }
+
+    public void assignRemoteControllerToTelevision(Long id, Long remoteControllerId) {
+        var optionalTelevision = repos.findById(id);
+        var optionalRemoteController = remoteControllerRepository.findById(remoteControllerId);
+
+        if(optionalTelevision.isPresent() && optionalRemoteController.isPresent()) {
```

Comment:
Let er wel op dat je nu niet makkelijk kan zienof er een probleem is met television of een probleem met remotecontroller